Altis_Life.Altis/initPlayerServer.sqf

```diff
@@ -1,10 +0,0 @@
-#include "script_macros.hpp"
-/*
-    File: initPlayerServer.sqf
-    Author:
-
-    Description:
-    Executed only on server when a player joins mission.
-*/
-if (LIFE_SETTINGS(getNumber,"player_deathLog") isEqualTo 0) exitWith {};
```

Comment:
We've lost whether to enable/disable it with your new fix.

Reply:
Done